sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFn.java

```diff
@@ -481,6 +481,12 @@ public Duration getAllowedTimestampSkew() {
     String value();
   }
 
+  /** Parameter annotation for the input element key for {@link OnTimer} methods. */
```

Comment:
I think should just be named Key, not KeyId.

Also I wouldn't reference OnTimer here, as @Key also makes sense in processElement (instead of getting a KV you should be able to get they key and the value in separate parameters.

Reply:
Done